config/webpack.config.js

```diff
@@ -14,15 +12,12 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const safePostCssParser = require('postcss-safe-parser');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
-const WorkboxWebpackPlugin = require('workbox-webpack-plugin');
```

Comment:
this has been removed, but is still in package.json, do we need this?

Reply:
![image](https://user-images.githubusercontent.com/1477453/82493154-26159a80-9af0-11ea-9f7c-4be58b24ecbb.png)

Reply:
May be for offline service workers, but lets remove it for now.

Reply:
Done